### ISSUE TITLE
Chocolatey install script fails - bad signature

### ISSUE THREAD

**Issue by alistairmgreen** [chocolatey/choco#1268]
This looks like a resurgence of issues #1203 and #1225.

When I run:
```
iwr https://chocolatey.org/install.ps1 -UseBasicParsing | iex
```

Chocolatey fails to install with [a long list of errors](https://gist.github.com/alistairmgreen/d9ac5070fb1dfb699e15ea03e37450d2) including:

```
Import-Module : File C:\ProgramData\chocolatey\helpers\chocolateyInstaller.psm1 cannot be loaded. The contents of file
C:\ProgramData\chocolatey\helpers\chocolateyInstaller.psm1 might have been changed by an unauthorized user or process,
because the hash of the file does not match the hash stored in the digital signature. The script cannot run on the
specified system. For more information, run Get-Help about_Signing..
```

* Operating system is Windows 7 Enterprise SP1.
* Problem first observed with PowerShell 2, but persists after upgrading to version 5.1.
* Execution policy is set to `AllSigned`.
* The script is trying to install Chocolatey version 0.10.5.

**Comment by ferventcoder** [comment]
Hi @alistairmgreen - it's #1225 and that fix has not yet been deployed (note 0.10.6 is the milestone), which makes this a duplicate of that issue.

**Comment by ferventcoder** [comment]
No resurgence, just never deployed yet. I know the docs say you can use AllSigned - we meant to get out a deployment of 0.10.6 as soon as possible, so we didn't update the docs. HTH
